### DISCUSSION TITLE
Is there any reason cookieconsent doesn't work on localhost?

### DISCUSSION THREAD

**Discussion by Anthony5485** [orestbida/cookieconsent#480] [Q&A]
I got it working without any problem on the production server but on my local one (https://localhost)  it doesn't show up. Any reasons why?

**Comment by orestbida** [comment]
Mhm, not really. Are there any errors/warning in console?

Please provide a minimal setup example.

**Reply by Anthony5485** [reply]
There was nothing in the console (while using localhost) which confused me a bit. When I uploaded the files on the production server (without changing anything else), it worked straightaway.
Here is the live website: https://www.europlayers.com
For info, I'm using Chrome on Windows 10.

**Reply by dombyte** [reply]
Maybe some issues with relative/static links. Have you checkt if the JS files alte loaded?
What Webserver do you use on your Win PC.
The Local Live Server of VS Code worked for me.

**Comment by dombyte** [comment]
I had the same issue when just opening the html page in the browser. For working properly I needed to use a web server. Otherwise your browser could block the cookie banner, for example brave does this. Then it's recommended to turn of the security features and try again. Or maybe a other browser works.

**Comment by orestbida** [comment]
@Anthony5485

there's nothing wrong with the config. Have you tried incognito mode or a different browser? Can you verify that the script is actually executed and that the html markup is generated?

**Comment by mkrawczykowski** [comment]
I had the same issue: no consent modal on localhost and Chrome. But opening it in incognito mode solves the problem.
